code/modules/clothing/spacesuits/rig/modules/ds13/kinesis/kinesis.dm

```diff
@@ -433,7 +433,8 @@
 //The default entrypoint, a wrapper for release
 //If the item is not in control range, it will be thrown based upon its velocity
 /obj/item/rig_module/kinesis/proc/release_grip()
-	var/speed = velocity.Magnitude()
+	//Precondition: Velocity must exist.
```

Comment:
this seems a bit sloppy. I'm sure it'll solve the immediate issue, but I'd like to know how velocity was null here in the first place, and prevent it from ever being so

Reply:
I can allocate more time to this if you'd prefer, and try find the cause. That being said though, this shouldn't be noticeable and ought to just result in a harsh stop rather than a smooth one.

Reply:
yes, this needs more work, find where and how velocity could be null

Reply:
From a play-session of around 30 mins, throwing stuff around and generally trying to break things, I was unable to reproduce this error.

Does this crop up a lot in rounds? If not, I think it's safe to apply this fix here, as it'll still maintain the throw behaviour, just losing the behaviour of it launching fast.